<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerViewer</class>
 <widget class="QWidget" name="LayerViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>multiTRACE Layer</string>
  </property>
  <widget class="QSpinBox" name="inputRow">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Row</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Column</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputCol">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addInput">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>105</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Input</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>172</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Strength, range: [0,1]</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inputStrength">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>57</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="layerImg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Here</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
